packages/datagrid/src/types/types.d.ts

```diff
@@ -4,3 +4,6 @@ declare module '*.scss' {
 }
 
 declare module '@talend/react-bootstrap';
+declare module '@talend/react-components' {
+	export * from '@talend/react-components/types';
+}
```

Comment:
without it, it do not compile. But it means you can just declare it as empty if you want